test/expected/drop_schema.out

```diff
@@ -0,0 +1,75 @@
+\c single :ROLE_SUPERUSER
+CREATE SCHEMA chunk_schema;
+CREATE SCHEMA hypertable_schema;
+CREATE SCHEMA extra_schema;
+GRANT ALL ON SCHEMA hypertable_schema TO :ROLE_DEFAULT_PERM_USER;
+GRANT ALL ON SCHEMA chunk_schema TO :ROLE_DEFAULT_PERM_USER;
+SET ROLE :ROLE_DEFAULT_PERM_USER;
+CREATE TABLE hypertable_schema.test (time timestamptz, temp float, location int);
+SELECT create_hypertable('hypertable_schema.test', 'time', 'location', 2, associated_schema_name => 'chunk_schema');
+NOTICE:  adding NOT NULL constraint to column "time"
+ create_hypertable 
+-------------------
+ 
+(1 row)
+
+INSERT INTO hypertable_schema.test VALUES ('2001-01-01 01:01:01', 23.3, 1);
+SELECT * FROM _timescaledb_catalog.hypertable;
+ id |    schema_name    | table_name | associated_schema_name | associated_table_prefix | num_dimensions 
```

Comment:
Can we also have a hypertable with an associated schema that isn't dropped so that we can see it doesn't change?